### ISSUE TITLE
Using Neumann BC with EB results in "Bottom solve failed."

### ISSUE THREAD

**Issue by RemiLehe** [ECP-WarpX/WarpX#3429]
When modifying our `ElectrostaticSphereEB` example to use Neumann boundary conditions instead of Dirichlet boundary conditions, the solver converges slowly and often prints  
```
MLMG: Bottom solve failed.
```
This is a bit surprising since it seems that this problem has a trivial solution: `phi = 1` everywhere in space.

Here is the input script: 
[inputs_3d.txt](https://github.com/ECP-WarpX/WarpX/files/9638914/inputs_3d.txt)
and here is the full output:
```
MLMG: # of AMR levels: 1
      # of MG levels on the coarsest AMR level: 5
MLMG: Initial rhs               = 0
MLMG: Initial residual (resid0) = 22162.18556
MLMG: Bottom solve failed.
MLMG: Iteration   1 Fine resid/resid0 = 0.02973627616
MLMG: Bottom solve failed.
MLMG: Iteration   2 Fine resid/resid0 = 0.002634105874
MLMG: Bottom solve failed.
MLMG: Iteration   3 Fine resid/resid0 = 0.0002298546815
MLMG: Bottom solve failed.
MLMG: Iteration   4 Fine resid/resid0 = 1.979884933e-05
MLMG: Bottom solve failed.
MLMG: Iteration   5 Fine resid/resid0 = 1.69703311e-06
MLMG: Bottom solve failed.
MLMG: Iteration   6 Fine resid/resid0 = 1.453122466e-07
MLMG: Bottom solve failed.
MLMG: Iteration   7 Fine resid/resid0 = 1.244156088e-08
MLMG: Final Iter. 7 resid, resid/resid0 = 0.0002757321809, 1.244156088e-08
MLMG: Timers: Solve = 0.30764974 Iter = 0.293613974 Bottom = 0.047949925
--- INFO    : Writing plotfile diags/diag1000000
```

**Comment by RemiLehe** [comment]
Here is how we set the boundary condition:
https://github.com/ECP-WarpX/WarpX/blob/development/Source/ablastr/fields/PoissonSolver.H#L255
https://github.com/ECP-WarpX/WarpX/blob/development/Source/FieldSolver/ElectrostaticSolver.cpp#L710

**Comment by WeiqunZhang** [comment]
This is an issue in amrex.  A fix will be submitted soon.

**Comment by ax3l** [comment]
Fix incoming via #3412 :)
